ci/pipelines/apps-test-ecr.yml

```diff
@@ -217,6 +234,9 @@ groups:
   - name: toolbox
     jobs:
       - toolbox-image-to-test-ecr
+      - deploy-toolbox
+      - smoke-test-toolbox
+      - push-toolbox-to-staging-ecr
```

Comment:
I think this definitely looks more practical in terms of stages we'd potentially want to re-run if there was a failure 👍